DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests.cs

```diff
@@ -26,38 +27,48 @@ public void Dispose()
         }
 
         [Theory]
-        [InlineData("/Home/Welcome")]
-        [InlineData("/Home/Products")]
-        [InlineData("/Home/LearningContent")]
-        [InlineData("/Login")]
-        [InlineData("/ForgotPassword")]
-        [InlineData("/ResetPassword/Error")]
-        public void Page_has_no_accessibility_errors(string url)
+        [InlineData("/Home/Welcome", "Welcome - Digital Learning Solutions")]
+        [InlineData("/Home/Products", "Products - Digital Learning Solutions")]
+        [InlineData("/Home/LearningContent", "Learning Content - Digital Learning Solutions")]
+        [InlineData("/Login", "Log in")]
+        [InlineData("/ForgotPassword", "Reset your password")]
+        [InlineData("/ResetPassword/Error", "Something went wrong...")]
+        public void Page_has_no_accessibility_errors(string url, string pageTitle)
         {
             // when
             driver.Navigate().GoToUrl(baseUrl + url);
             var axeResult = new AxeBuilder(driver).Analyze();
+            var h1Element = driver.FindElement(By.TagName("h1"));
 
             // then
-            axeResult.Violations.Should().BeEmpty();
+            using (new AssertionScope())
+            {
+                axeResult.Violations.Should().BeEmpty();
+                h1Element.Text.Should().BeEquivalentTo(pageTitle);
+            }

```

Comment:
I don't think we need to use the AssertionScope here, because the logic should be something like this:
```
// when 
navigate to url

// then
var h1Element = ...
h1Element.Text.Should().BeEquivalentTo(pageTitle);

var axeResult = ...
axeResult.Violations.Should().BeEmpty();
```

(No need to do the accessibility scans if we know that we're not on the correct page)

Reply:
You could even pull these two assertions into private methods that both tests can call, but that's not too important :)